support-frontend/assets/pages/supporter-plus-landing/supporterPlusRouter.tsx

```diff
@@ -47,6 +47,37 @@ const isInTwoStepTest =
 const showCheckoutTopUpToggle =
 	abParticipations.twoStepCheckoutWithNudgeBelow === 'variant_b';
 
+const checkCanMakePaymentsWithActiveCard =
+	abParticipations.canMakePaymentsWithActiveCard &&
+	abParticipations.canMakePaymentsWithActiveCard == 'control';
+
+/**
+ * Temporary check behind disabled AB test to validate whether
+ * window.ApplePaySession.canMakePaymentsWithActiveCard works
+ * on support.theguardian.com
+ */
+if (checkCanMakePaymentsWithActiveCard) {
+	const merchantIdentifier = 'merchant.uk.co.guardian.contributions';
+
+	const canMakePaymentsWithActiveCard = (): Promise<boolean> => {
+		return new Promise((resolve) => {
+			if (window.ApplePaySession) {
```

Comment:
Am getting Property 'ApplePaySession' does not exist on type 'Window & typeof globalThis'.ts(2339) error here. Can see its declared below however.